collector/metadata/kubernetes/pod_watch.go

```diff
@@ -29,7 +29,7 @@ type PodInfo struct {
 type podMap struct {
 	// namespace:
 	//   podName: podInfo{}
-	info  map[string]map[string]*PodInfo
+	Info  map[string]map[string]*PodInfo
```

Comment:
What is the purpose of modifying the access rights of this variable?

Reply:
Not really necessary, but want to be consistent with other maps, like the `serviceMap`. A case where this variable needs to be public is to marshall it to see its content in the unit test file.